examples/stable_diffusion_v2/ldm/modules/train/optim.py

```diff
@@ -43,14 +43,17 @@ def build_optimizer(
         _logger.info("Applying `%s` strategy for weight decay.", group_strategy)
 
     def decay_filter(param):
-        if group_strategy is not None and group_strategy.lower() == "unclip":
+        if group_strategy is None:
+            filter_list = ["layernorm", "bias"]
+        elif group_strategy.lower() == "unclip":
```

Comment:
Do we need to update `mindone/trainers/optim.py` as well?

Reply:
Updated. Thx for remind